app/templates/layout.html

```diff
@@ -78,5 +78,6 @@
 
     <script src="{{ basepath }}/components/jquery/jquery.js"></script>
     <script src="{{ basepath }}/components/bootstrap/bootstrap-built.js"></script>
+    <script src="{{ basepath }}/js/equalHeightColumns.js"></script>
```

Comment:
Conditional is like this:

    {% if page == 'department' %}stuff here{% endif %}